xmodule/x_module.py

```diff
@@ -1656,15 +1656,15 @@ class ModuleSystem(MetricsMixin, ConfigurableFragmentWrapper, ModuleSystemShim,
 
     def __init__(
         self,
-        get_module,
+        get_block,
         descriptor_runtime,
-        **kwargs,
+        **kwargs
```

Comment:
```suggestion
        **kwargs,
```

Nit: please keep trailing commas. Even though we're not going to add anything after `**kwargs` (so it doesn't matter in the context of producing a shorter `git diff`), it keeps the code consistent.